relm4-components/Cargo.toml

```diff
@@ -27,12 +27,18 @@ once_cell = "1.18"
 relm4 = { version = "0.7.0-alpha.1", path = "../relm4", default-features = false, features = ["macros"] }
 reqwest = { version = "0.11.18", optional = true }
 tracker = "0.2.1"
+adw = { version = "0.5", optional = true, package = "libadwaita" }
 
 [features]
-default = []
+default = ["libadwaita"]
 dox = ["web"]
 web = ["reqwest"]
+libadwaita = ["adw", "relm4/libadwaita"]
```

Comment:
`libadwaita = ["relm4/libadwaita"]` is enough, you can import adwaita through `relm4::adw`.

Reply:
Right!

Fixed it.